src/main/resources/treeformats.yml

```diff
@@ -51,6 +51,17 @@ formats:
     version:
       regex: "@([^@\\s]+)(\\s\\S+)?$"
       group: 1
+  - format: nuget
+    description: "Nuget tree output"
+    tool: "See https://github.com/Brend-Smits/NugetTree"
+    types:
+      "": "nuget"
+    name:
+      regex: "([^ ]*)"
+      group: 1
+    version:
+      regex: "(?<= )(\\d.+)"
```

Comment:
Looking at the sample tree output, I would expect some special tricks to handle e.g. `xunit.extensibility.core 2.4.1.0 by xunit.core (2.4.1.0)`.

If the separator character between name and version is always just a space, then the name might be matched by e.g. `(^\\S+)` and the version by `\\s(\\S+)`. (Both take group 1 as their result.)

Reply:
Thanks for the suggestion, this works really well!